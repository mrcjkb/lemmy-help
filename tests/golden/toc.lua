---@toc my-plugin.contents

---@mod first.module First Module

---@mod second.module Second Module

---@mod third.module Third Module

local U = {}

return U
