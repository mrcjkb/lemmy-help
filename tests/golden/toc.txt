==============================================================================
Table of Contents                                           *my-plugin.contents*

First Module ···················································· |first.module|
Second Module ·················································· |second.module|
Third Module ···················································· |third.module|

==============================================================================
First Module                                                      *first.module*

==============================================================================
Second Module                                                    *second.module*

==============================================================================
Third Module                                                      *third.module*


